pandas/io/json/_json.py

```diff
@@ -894,14 +895,19 @@ def _try_convert_data(self, name, data, use_dtypes=True, convert_dates=True):
             if result:
                 return new_data, True
 
-        result = False
+        result = True
 
         if data.dtype == "object":
-
+            if (
+                isinstance(data, Index)
```

Comment:
i don't think you need to test if an Index

Reply:
Done